client/a8c-for-agencies/data/team/use-fetch-active-members.ts

```diff
@@ -0,0 +1,30 @@
+import { UseQueryResult, useQuery } from '@tanstack/react-query';
+import { TeamMember } from 'calypso/a8c-for-agencies/sections/team/types';
+import wpcom from 'calypso/lib/wp';
+import { useSelector } from 'calypso/state';
+import { getActiveAgencyId } from 'calypso/state/a8c-for-agencies/agency/selectors';
+
+export default function useFetchActiveMembers(): UseQueryResult< TeamMember[], unknown > {
+	const agencyId = useSelector( getActiveAgencyId );
+
+	return useQuery( {
+		queryKey: [ 'a4a-active-members', agencyId ],
+		queryFn: () =>
+			wpcom.req.get( {
+				apiNamespace: 'wpcom/v2',
+				path: `/agency/${ agencyId }/users`,
+			} ),
+		select: ( data ) => {
+			// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Are we going to update this type once the API is finalized?

Reply:
This has been addressed. I've added a type for this.